packages/frontend/app/models/vote-chain.ts

```diff
@@ -9,14 +11,17 @@ export enum VOTE_ACTION {
 }
 
 export default class VoteChain extends Model {
-  @hasMany('identity', { async: false }) remaining!: Identity[];
-  @hasMany('identity', { async: false }) yes!: Identity[];
-  @hasMany('identity', { async: false }) no!: Identity[];
-  @attr() target!: Identity;
+  @hasMany('identity', { async: false, inverse: 'stillRemainingIn' }) remaining!: Identity[];
+  @hasMany('identity', { async: false, inverse: 'votedYesIn' }) yes!: Identity[];
+  @hasMany('identity', { async: false, inverse: 'votedNoIn' }) no!: Identity[];
+  @belongsTo('identity', { async: false, inverse: 'targetOfVote' }) target!: Identity;
   @attr() action!: VOTE_ACTION;
-  @attr() key!: Identity;
-  @attr() previousVoteChain?: VoteChain;
+  @belongsTo('identity', { async: false, inverse: 'voterOf' }) key!: Identity;
+  @belongsTo('vote-chain', { async: false, inverse: 'parentChain' }) previousVoteChain!: VoteChain;
+  @belongsTo('vote-chain', { async: true, inverse: 'previousVoteChain' }) parentChain!: VoteChain;
```

Comment:
why async: true?

Reply:
Same thing here

Reply:
Essentially we put these on all the back relationships that were non-existent before because there isn't really any use for them outside of structuring the records properly.

Reply:
Addressed in d005d5f